### DISCUSSION TITLE
Colour inconsistencies

### DISCUSSION THREAD

**Discussion by Teloriun** [openscopeproject/TrguiNG#55] [General]
First off - great app, came across it this week and it's brilliant so far. Thank you.

Regarding the topic, it's not an issue as such, but more of an observation (sorry).

eg.  
you use red x for error'd torrents on left status and main window (which match), but the same symbol is used for remove/delete on top toolbar (maybe should be wastebin instead?)
pause is yellow on left status, green tick on main window and blue on toolbar (maybe they should all be the yellow pause symbol as green is used for upload and blue for download?)

![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/e780a3e4-7086-4799-840b-7f72574ea355)
![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/a6326e35-1548-4c0e-be0c-6466008c802c)
![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/34d32a3a-eba5-450d-9cf0-c0641a891060)

![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/9b0c3282-d8ed-45a3-ad82-45998a8a1326)
![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/f6c7c2b8-e83e-4fd2-abb8-9e25be131f7e)
![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/a56b3803-2efd-419f-a089-deb4edcc16f8)

Running on Windows 10 Pro / TrguiNG_0.8.1_x64-setup.exe

**Comment by qu1ck** [comment]
The different shades of red are because contrasting rich red is necessary on small icons to maintain good visibility when corresponding row is selected and has blue background. The toolbar does not have that background and does not need strong red shade.
But I made the selection blue darker at a later time so the shade of red in tables can probably be reconsidered.

The paused state actually has different sub states for paused incomplete (yellow pause icon) and paused complete (green checkmark). The toolbar button should be yellow too in theory but it does not look good in white theme. I'm not happy with the yellow priority icon either, the inner details are not discernible.

**Reply by Teloriun** [reply]
I think the colour is fine, it's more to do with the icon for delete being the same as the icon for error, an x symbol.

Thanks for explaining the sub states.  I also changed to light mode to view the yellow icon and can see your point of view.  Apart from changing the colour (which you might still do as you don't sound happy with it), maybe an alternative would be to change the symbol within to a square stop sign rather than a pause (as you've called the status 'Stopped')?  eg.![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/024e755b-6dfc-4067-910e-12211c5d5bba)![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/d9885ee9-7c5e-4dd3-baf0-a4a5751c3b4c)

That would probably mean changing the top toolbar icon to a square symbol as well rather than a pause symbol for consistency.
eg.![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/fce9c853-c4b6-4732-a001-eb4d2381b517)![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/24879640-07a2-4804-8a88-357df8d73ce4)

  Alternatively, stretch the box a little - from the naked eye it looks 'smaller' in height than the other status symbols so might make a difference? 

Again, only throwing ideas.  Feel free to ignore - it's your project.. I'm just glad you've shared it :)

**Reply by qu1ck** [reply]
Yeah, the icons need some work and this was brought up earlier too. I deprioritized it before in favor of features and functionality but now that slowed down significantly and maybe it's time to dust off inkscape and draw icons.
